packs/spell-effects/spell-effect-albatross-curse-critical-failure.json

```diff
@@ -0,0 +1,42 @@
+{
+    "_id": "jSvpjSGnIVBAuFDu",
+    "img": "icons/creatures/birds/dove-pigeon-flying-white.webp",
+    "name": "Spell Effect: Albatross Curse (Critical Failure)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Albatross Curse].</p>\n<p>The albatross hangs around a cord from the target’s neck (or closest equivalent) for 1 hour, cursing them for their transgression. During this time, the target must roll twice and take the worse result on their next Will save, after which the albatross disappears.</p>"
```

Comment:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Albatross Curse].</p>\n<p>You must roll twice and take the worse result on your next Will save.</p>"
```

Reply:
I thought of writing it this way initially, but checked existing effects to see what the style was.

Every effect applied to a spell target continues to use the text, verbatim or nearly so, from the spell description.  This refers to the target in the 3rd person.  The text isn't altered to refer to the target in the 2nd person, despite the effect being on the target.  Some examples:

- Practice Makes Perfect: The target gains a +2 status bonus…
- Spiral of Horrors: Foes in the area are frightened 1…
- Faerie Dust: The creature takes a -1 status penalty…
- Repel Metal: The target gains a +2 circumstance bonus…

Every spell effect using "you" does so to refer to the caster of the spell or, in some cases, the caster and their allies.  At least after checking several dozen spell effects, they have all been consistent in this so far.

Reply:
I believe we're going with 2nd person on effects going forward, but one of the leads can decide if it's necessary. Regardless, it will need trimming down to only the implemented mechanical effects, i.e. the roll twice keep lower on will.

Reply:
So far the effects I've checked have been entirely consistent, 2nd person caster, 3rd person target.  Surprising really, with all the different people doing data entry, that there isn't a more of a mix of point of view, tenses, voices, and so on.  I suspect it's because they copy Paizo's text with a minimum of editing.  Re-writing the rules will probably lead to a lot less consistency.

And of course once the rules get reworded, there can be arguments about how the rewording changes the meaning.  Data entry can become a rules thread.

Reply:
From the [Style Guide](https://github.com/foundryvtt/pf2e/wiki/Style-Guide#effects):
The main description of an effect should, in general, only say _exactly_ what the effect actually automates. Do not include things the effect item does not do, and do not include the duration in the description. For example, a spell effect granting a 20 foot fly speed and Quickened that can only be used to fly would have an effect with a main description of `You gain a 20-foot fly speed and become Quickened.`

Reply:
I'd already trimmed it to just the effect that was automated, but I left in the duration.  I've removed that now too.